packages/image_picker/image_picker_android/example/lib/main.dart

```diff
@@ -10,6 +10,7 @@ import 'dart:io';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_driver/driver_extension.dart';
+import 'package:image_picker_android/image_picker_android.dart';
 import 'package:image_picker_platform_interface/image_picker_platform_interface.dart';
```

Comment:
We should wrap these two lines in `// #docregion photo-picker-example`/// #enddocregion photo-picker-example` too so that they show up in the snippet, otherwise people may be confused about where to get these APIs; the pattern of depending on non-app-facing packages is still pretty uncommon.